images/fcos-bastion-image/root/usr/bin/clean-registries.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+set -x
+
+for port in 5000 6001 6002; do
```

Comment:
Hi @jadhaj , this repo should not have knowledge about which registries unit are enabled. Can we run something to check which templated units are enabled?